src/cc_helpers.cc

```diff
@@ -82,16 +82,22 @@ extern string gramine_platform_cert;
 
 certifier::framework::cc_trust_data::cc_trust_data(const string& enclave_type, const string& purpose,
     const string& policy_store_name) {
-
+  cc_trust_data_default_init();
```

Comment:
Rearrange the constructor to do the "default initialization" first, followed by the init's needed by passed-in args.

Reply:
OK